src/store.js

```diff
@@ -165,6 +166,9 @@ export default new Vuex.Store({
 		cleanStateFor(state, path) {
 			Vue.delete(state.vState, path)
 		},
+		setLanguages(state, payload) {
+			state.languages = Object.assign({}, state.languages, payload)
```

Comment:
this will not work with IE, deep copy would ensure that.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign

Reply:
@jppiiroinen: Our package.json browserlist rules currently include IE 10 and 11 (try `npx browserslist` in qvain-js for a list), which means that our default Babel preset includes a Object.assign polyfill. We might use some other features that won't work in IE with our current configuration, but Object.assign should work.
https://cli.vuejs.org/guide/browser-compatibility.html
https://github.com/vuejs/vue-cli/tree/dev/packages/%40vue/babel-preset-app

Reply:
ok